firestore/integration_test_internal/src/bundle_test.cc

```diff
@@ -1,5 +1,6 @@
 // Copyright 2021 Google LLC
 
+// The following line should cause a lint warning.
 #include <future>
```

Comment:
⚠️ Lint warning: `<future> is an unapproved C++11 header.`<hidden value="cpplint-file-comment"></hidden><hidden value="firestore/integration_test_internal/src/bundle_test.cc:4:  <future> is an unapproved C++11 header.  [build/c++11] [5]"></hidden>